app/models/version.rb

```diff
@@ -171,6 +171,10 @@ def platformed?
 
   delegate :reorder_versions, to: :rubygem
 
+  def can_yank
```

Comment:
The reason I didn't use `yankable` is because it isn't an actual word by dictionary.